pages/volunteersignup.tsx

```diff
@@ -0,0 +1,494 @@
+import {
+    Button,
+    Box,
+    Center,
+    Flex,
+    Text,
+    Input,
+    FormControl,
+    FormLabel,
+    Link,
+    Progress,
+    Stack,
+    HStack,
+    VStack,
+    Select,
+    Checkbox,
+    Textarea,
+    Heading,
+    UnorderedList,
+    ListItem,
+    OrderedList,
+    Radio,
+    RadioGroup,
+} from "@chakra-ui/react";
+import { CloseButton } from "@components/CloseButton";
+import { BackButton } from "@components/BackButton";
+import { useState } from "react";
+import { GetServerSideProps } from "next"; // Get server side props
+import { getSession, GetSessionOptions } from "next-auth/client";
+import Wrapper from "@components/SDCWrapper";
+import useLocalStorage from "@utils/useLocalStorage";
+import { roles, locale, province, VolunteerInput } from "@models/User";
+
+const BLUE = "#0C53A0"; // TODO: move to src/styles
+const RADIO_YES = "yes";
+const RADIO_NO = "no";
+const DEFAULT_PROVINCE = province.BC;
+
+const FormButton = (props) => {
+    return (
+        <Button
+            bg={BLUE}
+            color={"white"}
+            fontWeight="400"
+            onClick={props.onClick}
+            my={8}
+            px={12}
+            borderRadius={100}
+        >
+            {props.children}
+        </Button>
+    );
+};
+
+const FormPage = (props) => {
+    return <Stack spacing={8}>{props.children}</Stack>;
+};
+
+/**
+ * This is the page that a volunteer will use to enter the volunteers personal information
+ * onto the SDC platform as a volunteer
+ */
+export default function VolunteerInfo({
+    session,
+}: {
+    session: Record<string, unknown>;
+}): JSX.Element {
+    const [progressBar, setProgressBar] = useState(Number);
+    const [pageNum, setPageNum] = useState(0);
+    const formButtonOnClick = () => {
+        setPageNum((prevPage) => prevPage + 1);
+        window.scrollTo({ top: 0 });
+    };
+
+    /* Store form fields in local storage */
+    //Volunteer Info
+
+    const [volunteerFirstName, setVolunteerFirstName] = useLocalStorage(
+        "volunteerFirstName",
+        "",
+    );
+    const [volunteerLastName, setVolunteerLastName] = useLocalStorage(
+        "volunteerLastName",
+        "",
+    );
+    const [phoneNumber, setPhoneNumber] = useLocalStorage("phoneNumber", "");
+
+    //volunteer Personal Details
+
+    const [dateOfBirth, setDateOfBirth] = useLocalStorage("dateOfBirth", "");
+    const [address1, setAddress1] = useLocalStorage("address1", "");
+    const [city, setCity] = useLocalStorage("city", "");
+    const [participantProvince, setParticipantProvince] =
+        useState(DEFAULT_PROVINCE);
+    const [postalCode, setPostalCode] = useLocalStorage("postalCode", "");
+    const [school, setSchool] = useLocalStorage("school", "");
+
+    //volunteer personal details
+
+    const [skills, setSkills] = useLocalStorage("skills", "");
+    const [heardFrom, setHeardFrom] = useLocalStorage("heardFrom", "");
+    const [attending, setAttending] = useState(false);
```

Comment:
these don't look like they're used - should they be used somewhere? or can we get rid of them?

Reply:
Wait im not sure which one your talking about but Skills is used on line 238-239, heard from is used under that on lines 249-250. And I took out attending

Reply:
I was just referring to attending